cpp-client/deephaven/client/include/private/deephaven/client/impl/table_handle_impl.h

```diff
@@ -37,11 +37,8 @@ class TableHandleManagerImpl;
 namespace internal {
 class GetColumnDefsCallback;
 
-class LazyState final
+class EtcCallback final
```

Comment:
Is Etc == ExportedTableCreation? :-)
I am all in for terseness, but it sounds a bit funny to me to abbrev to "etc" and then spell out "Callback" in full :-P.
No action item here, not asking you to change anything, just thinking out loud.  Maybe put a comment on top.